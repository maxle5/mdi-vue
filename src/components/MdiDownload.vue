
<template>
  <svg :width="size" :height="size" :viewBox="viewBox">
      <path d="M5,20H19V18H5M19,9H15V3H9V9H5L12,16L19,9Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiDownload",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
</script>
