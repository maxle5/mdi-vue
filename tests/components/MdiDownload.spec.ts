
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiDownload from "../../src/components/MdiDownload.vue";

test("MdiDownload snapshot", () => {
  const wrapper = mount(MdiDownload, {});
  expect(wrapper.html()).toMatchSnapshot();
});
