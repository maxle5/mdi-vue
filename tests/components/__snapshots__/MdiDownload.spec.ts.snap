// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiDownload snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M5,20H19V18H5M19,9H15V3H9V9H5L12,16L19,9Z" style="fill: currentColor;"></path>
</svg>"
`;
